aws_pricing/pyproject.toml

```diff
@@ -18,28 +18,27 @@ keywords = [
 ]
 authors = [
     { email = "tsein@brightcove.com" },
+    { name = "Devin Young", email = "dyoung@pagerduty.com" },
 ]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
     "Intended Audience :: System Administrators",
     "License :: OSI Approved :: BSD License",
-    "Programming Language :: Python :: 2.7",
-    "Programming Language :: Python :: 3.8",
-    "Topic :: System :: Monitoring",
     "Private :: Do Not Upload",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: System :: Monitoring",
 ]
 dependencies = [
-    "datadog-checks-base>=4.2.0",
+    "boto3>=1.28.0",
```

Comment:
Not a big deal because of how the Agent installs integration, but we generally prefer to keep 3rd party dependencies under `project.optional-dependencies` (just referring to `boto3` in this case).